compiler-plugin-tests/src/test/resources/ballerina_sources/invalid_content_service_22/Ballerina.toml

```diff
@@ -0,0 +1,4 @@
+[package]
+org = "ftp_test"
+name = "invalid_content_service_22"
+version = "0.1.0"
```

Comment:
```suggestion
version = "0.1.0"

```

Reply:
Fixed in [621e44c](https://github.com/ballerina-platform/module-ballerina-ftp/pull/1517/commits/621e44cba6f91159d27da515c1884c26bc38f38d)